src/js/constants/UnitSummaries.js

```diff
@@ -1,156 +1,210 @@
+    summary: [
+      i18nMark(
+        "Restart the Admin Router Nginx server so that it picks up new DNS resolutions, for example master.mesos, leader.mesos."
+      )
+    ],
     getDocumentationURI() {
       return MetadataStore.buildDocsURI("/");
     }
   },
   "dcos-nginx-reload.timer": {
-    summary:
-      "Sets the dcos-adminrouter-reload.service interval at once an hour.",
+    summary: [
+      i18nMark(
+        "Sets the dcos-adminrouter-reload.service interval at once an hour."
+      )
+    ],
     getDocumentationURI() {
       return MetadataStore.buildDocsURI("/");
     }
   },
   "dcos-setup.service": {
-    summary: `Specializes the ${
-      Config.productName
-    } bootstrap tarball for the particular cluster, as well as its cluster role.`,
+    summary: [
+      i18nMark("Specializes the"),
+      Config.productName,
+      i18nMark(
+        "bootstrap tarball for the particular cluster, as well as its cluster role."
+      )
+    ],
     getDocumentationURI() {
       return MetadataStore.buildDocsURI("/");
     }
   },
   "dcos-mesos-slave.service": {
-    summary: "Runs a Mesos agent on the node.",
+    summary: [i18nMark("Runs a Mesos agent on the node.")],
     getDocumentationURI() {
       return MetadataStore.buildDocsURI("/");
     }
   },
   "dcos-keepalived.service": {
-    summary:
-      "Runs keepalived to make a VRRP load balancer that can be used to access the masters.",
+    summary: [
+      i18nMark(
+        "Runs keepalived to make a VRRP load balancer that can be used to access the masters."
+      )
+    ],
     getDocumentationURI() {
       return MetadataStore.buildDocsURI("/");
     }
   },
-  "dcos-adminrouter.service": {
-    summary: `Runs the ${
-      Config.productName
-    } web interface, as well as a reverse proxy so that administrative interfaces of ${
-      Config.productName
-    } Service can be accessed from outside the cluster.'`,
+  "dcos-adminrouter-agent.service": {
+    summary: [
+      i18nMark("Runs the"),
+      Config.productName,
+      i18nMark(
+        "web interface, as well as a reverse proxy so that administrative interfaces of"
+      ),
+      Config.productName,
+      i18nMark("Service can be accessed from outside the cluster.")
+    ],
     getDocumentationURI() {
       return MetadataStore.buildDocsURI("/");
     }
   },
   "dcos-gen-resolvconf.service": {
-    summary:
-      "Periodically writes /etc/resolv.conf so that only currently active Mesos masters with working Mesos DNS are in it.",
+    summary: [
+      i18nMark(
+        "Periodically writes /etc/resolv.conf so that only currently active Mesos masters with working Mesos DNS are in it."
+      )
+    ],
     getDocumentationURI() {
       return MetadataStore.buildDocsURI("/installing/ent/");
     }
   },
   "dcos-mesos-slave-public": {
-    summary: "Runs a publicly accessible Mesos agent on the node.",
+    summary: [i18nMark("Runs a publicly accessible Mesos agent on the node.")],
```

Comment:
Looking back at this file, I wonder if it would make a better translation to macro them all with tokens for productName inside, and just make summary a React node. We'd have to change a couple prop types to support node instead of string, but it would yield a less fragmented translation